lib/Charts/ChartType.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+import defined from 'terriajs-cesium/Source/Core/defined';
+
+import LineChart from './LineChart';
+import BarChart from './BarChart';
+
+const chartType = {
+    line: new LineChart(),
+    bar: new BarChart()
+};
+
+function determineChartType(chartData) {
+    
+    // respect chartData.type
+    if (chartType[chartData.type] !== undefined) {
+        return chartType[chartData.type];
+    }
+    
+    // insert hack here for figuring out which type of chart datasets need
+    try
```

Comment:
Did you have any suggestions on how to choose the chart type in light of my earlier comment?